packages/typespec-ts/test/integration/generated/sharedRoute/src/isUnexpected.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
why this is not generated before?

Reply:
i update the tsp, previously there is only one status code defined so no isUnexpected helper would generate.